app/views/users/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<h1>Sign Up</h1>
+<%= form_for @user,{method: "post", url: users_path} do |f| %>
```

Comment:
This will also work, and it's a bit shorter and less error prone:
```rb
<%= form_for @user do |f| %>
  ...
```
http://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html